### PR TITLE
Including rules tcp and upd where are the same

### PR DESCRIPTION
**Pull request by hmunfru**
#### Reviewers
@jframos @jesuspg

#### Description
 https://jirapdi.tid.es/browse/CLAUDIA-4851

#### Test
tested locally

### CODE REVIEW

File: model/src/main/java/com/telefonica/euro_iaas/paasmanager/model/Rule.java [hunk 1]
```diff
@@ -194,6 +194,9 @@ public boolean equals(Object obj) {
         if (this.idparent != null && !this.idparent.equals(other.idparent)) {
```
Comment (flopezag) [inline model/src/main/java/com/telefonica/euro_iaas/paasmanager/model/Rule.java]: The method equals(Object obj) cannot be accepted in the way that it is coded.
1) I do not understand the reason why are you sending an Object class and not a Rule class.
2) One method - one return
3) Add javadoc

**Comment by jesuspg** [comment]
LGTM

**Comment by flopezag** [comment]
Please take care about the update of the Rule.java class...

**Comment by hmunfru** [comment]
now this PR does no make sense. I will close it.

**Comment by jesuspg** [comment]
but, what about unit test!

**Comment by flopezag** [comment]
Yes, @jesuspg is correct, I assume that you create a new PR with this two unit test changed files.

**Pull request merged**